play-json/shared/src/main/scala/play/api/libs/json/JsLookup.scala

```diff
@@ -173,13 +156,37 @@ object JsLookupResult {
 /**
  * Wrapper for JsValue to represent an existing Json value.
  */
-case class JsDefined(value: JsValue) extends AnyVal with JsLookupResult
+case class JsDefined(value: JsValue) extends AnyVal with JsLookupResult {
+
+  override def toOption: Option[JsValue] = Some(value)
+
+  override def toEither: Either[JsonValidationError, JsValue] = Right(value)
+
+  override def isEmpty: Boolean = false
```

Comment:
`val`

Reply:
I'm not sure but this can't be done in a value class right ?

Reply:
there is no benefit to using a `val` here because the return value is already the constant `false`.